docs/trace/getting-started-jaeger/getting-started-jaeger.csproj

```diff
@@ -5,6 +5,6 @@
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Instrumentation.Http\OpenTelemetry.Instrumentation.Http.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.Net.Http" Version="4.3.4" Condition="$(TargetFramework) == 'net472' or $(TargetFramework) == 'net48'" />
```

Comment:
Red flag for me that the Condition was dropped.

Reply:
[fixed](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4298/commits/52c6481d73fce2ef28ff75c9091c47a787bc05fa)